inc/saitypes.h

```diff
@@ -264,7 +274,16 @@ typedef enum _sai_object_type_t
     SAI_OBJECT_TYPE_DEBUG_COUNTER            = 85,
     SAI_OBJECT_TYPE_PORT_CONNECTOR           = 86,
     SAI_OBJECT_TYPE_PORT_SERDES              = 87,
-    SAI_OBJECT_TYPE_MAX                      = 88,
+    SAI_OBJECT_TYPE_PORT_SERDES              = 88,
```

Comment:
duplicated enum here for the enum value 87 & 88 SAI_OBJECT_TYPE_MACSEC_PORT_SERDES for this line?